src/utils/numAbbr.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * @flow
+ * @file Function to abbreviate a number as a string in a locale-sensitive manner.
+ * @author Box
+ */
+
+import IntlMessageFormat from 'intl-messageformat';
+import { languages, numbers } from 'box-locale-data'; // eslint-disable-line
```

Comment:
How would this work for external platform customers?
Right now documentation suggests we do whats listed here https://github.com/formatjs/react-intl/blob/master/docs/Getting-Started.md#runtime-requirements

Reply:
This is how an external customer will use https://github.com/box/box-ui-elements-demo/blob/master/src/Main.js

Reply:
One way is that the cldr data should be published along with buie as an importable module?

Reply:
Yes, the @box/cldr-data module is importable separately. The way react-intl does it is have your app explicitly require the locale data for all the locales the app needs, so that they all make it into the webpacked result.

The way that our own locale data is included in box-ui-elements is similar to the way that the translated strings are included. That is, the 'box-locale-data' module is actually an alias that causes `node_modules/@box/cldr-data/es/locale-data/${language}` to be loaded instead. That way, only the locale data for the current locale is included in the webpacked result, instead of the locale data for all locales at once and picking the right one at run time. Helps reduce the disk/memory footprint!